### PR TITLE
talk to me

### LINKED ISSUE #49: (3) talk to me
## Description
`Level 3`
> add a fake live chatbox to the contact us page. the ai behind this will be highly sophisticated: whenever the user asks the bot a question it will always respond with "I didn't quite understand that, please try rephrasing what you said." you will need to create a chatbox component that is either a react stateful component, or uses the `useState` react hook. it should list out all the messages from the user and the "bot", have a text input for the user to type a message, and a send button that "sends" the message. this one is a bit complicated so feel free to ask lots of questions

### PR DESCRIPTION
**Pull request by foltik**
Fixes #49

### CODE REVIEW

**Comment by foltik** [comment]
Added to the "Contact us" page

File: challenge/src/components/Pages/Contact.jsx [hunk 1]
```diff
@@ -10,44 +11,47 @@ const Contact = (props) => {
...
-              <RandomImage
-                page={page * 8}
-                style={pictureStyling}
-              />
+              <h1 style={{ fontSize: 100, color: "#F09F70" }}>
+                <center>Contact Page</center>
+              </h1>
+              <div style={{ backgroundColor: "whitesmoke" }}>
+                <br />
+              </div>
```
Comment (mjrb) [inline challenge/src/components/Pages/Contact.jsx:22]: please avoid adding formatting changes or reindenting code to PRs that also include other stuff. it makes it hard to tell what things actually change during code review. this should be done in separate PRs

**Review by mjrb — APPROVED** [review]
LMFAO this is literally everything I could have hoped for, thank you

File: challenge/src/components/Pages/Contact.jsx (hunk 1 above)
Comment (foltik) [reply challenge/src/components/Pages/Contact.jsx:22]: Gotcha. Indented everything in a level because I wrapped everything with a `<>`.
